screensaver.unsplash/resources/skins/default/1080i/default.xml

```diff
@@ -33,11 +52,12 @@
             <shadowcolor>text_shadow</shadowcolor>
             <height>80</height>
             <width>auto</width>
-            <label>$INFO[System.Time]</label>
-            <animation effect="fade" start="25" end="25" condition="True">Conditional</animation>
+            <label>[B]$INFO[System.Time][/B]</label>
+            <animation effect="fade" start="15" end="15" condition="True">Conditional</animation>
+            <visible>StringCompare(Window.Property(unsplash_time),okay) + [Control.IsVisible(30000)|Control.IsVisible(30001)]</visible>
         </control>
-        <control type="label" id="30001">
-            <posx>1695</posx>
+        <control type="label">
+            <posx>1755</posx>
             <posy>930</posy>
             <description>Time</description>
             <align>left</align>
```

Comment:
Okay, THX... I wish there was a standard schema in place for font names, at least till scripts get some custom font love = )